sycl/include/sycl/ext/intel/esimd.hpp

```diff
@@ -0,0 +1,19 @@
+//==------------ esimd.hpp - DPC++ Explicit SIMD API redirection header ----==//
```

Comment:
why this file is added?
The vanilla location for esimd.hpp is `sycl/ext/intel/experimental/esimd.hpp`, why add one more redirection?

Reply:
@kbobrovs , from the description:
```
Copy
include/CL/sycl/ONEAPI to sycl/ext/oneapi and include/CL/sycl/INTEL to
sycl/ext/intel as is.
```

Reply:
anyway, sycl/include/sycl/ext/intel/esimd.hpp should be removed. include/CL/sycl/INTEL/esimd.h header redirection is already in-place, so there is no need to add any more files.

Reply:
Done